pyproject.toml

```diff
@@ -68,14 +66,16 @@ spyglass_cli = "spyglass.cli:cli"
 "Bug Tracker" = "https://github.com/LorenFrankLab/spyglass/issues"
 
 [project.optional-dependencies]
-position = ["ffmpeg", "numba>=0.54", "deeplabcut<2.3.0"]
+dlc = ["ffmpeg", "numba>=0.54", "deeplabcut<2.3.0"]
 test = [
+    "black[jupyter]", # code formatting, rather than a 'dev' install
```

Comment:
I do think it is handy for users in our lab to have black installed for our own sanity.

Reply:
From the PRs I've seen by irregular contributors, having it installed doesn't mean it gets run. Yes, it's there, but there's still a step of 'please run'. I can put it back if you feel strongly about it. The user I had in mind during these revisions was someone who is unlikely to ever contribute

Reply:
Let's just keep it for now. I still have hope that it is helpful for people in the lab.